spec/services/api/v1.2/catalog/get_linked_order_process_spec.rb

```diff
@@ -0,0 +1,82 @@
+describe Api::V1x2::Catalog::GetLinkedOrderProcess, :type => [:service] do
+  around do |example|
+    Insights::API::Common::Request.with_request(default_request) { example.call }
+  end
+
+  include_context "tag links of an order process with parameters set"
+
+  subject { described_class.new(params) }
+
+  shared_examples_for '#test_catalog_process' do
+    it 'gets linked order processes' do
+      with_modified_env test_env do
+        Api::V1x2::Catalog::LinkToOrderProcess.new(params).process
+
+        expect(subject.process.order_processes).to eq([order_process])
+      end
+    end
+  end
+
+  shared_examples_for '#test_remote_process' do
+    it 'gets linked order processes' do
+      stub_request(:get, url).to_return(:status => http_status, :body => {:data => order_process_tags}.to_json, :headers => headers)
+
+      with_modified_env test_env do
+        TagLink.create(:order_process_id => order_process.id.to_s,
+                       :object_type      => object_type,
+                       :app_name         => app_name,
+                       :tag_name         => order_process_tag[:tag])
+
+        expect(subject.process.order_processes).to eq([order_process])
+      end
+    end
+  end
+
+  describe 'catalog' do
+    let(:app_name) { 'catalog' }
+
+    context 'when object type is portfolio' do
+      let(:portfolio) { create(:portfolio) }
+      let(:object_id) { portfolio.id }
+      let(:object_type) { 'Portfolio' }
+      it_behaves_like "#test_catalog_process"
+    end
+
+    context 'when object type is portfolio_item' do
+      let(:portfolio_item) { create(:portfolio_item) }
+      let(:object_id) { portfolio_item.id }
+      let(:object_type) { 'PortfolioItem' }
+      it_behaves_like "#test_catalog_process"
+    end
+  end
+
+  describe 'topology' do
+    let(:object_id) { '123' }
+    let(:app_name) { 'topology' }
+    let(:env_not_set) { /TOPOLOGICAL_INVENTORY_URL is not set/ }
```

Comment:
I can't find anywhere that all of the declarations of `env_not_set` are actually being used, do we still need them?